library/src/main/java/com/schibsted/spain/barista/custom/ClickChildAction.java

```diff
@@ -0,0 +1,61 @@
+package com.schibsted.spain.barista.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import android.widget.TextView;
+
+import com.schibsted.spain.barista.util.ViewTreeAnalyzer;
+
+import org.hamcrest.Matcher;
+
+public class ClickChildAction {
+
+  public static ViewAction clickChildWithId(final int id) {
```

Comment:
Suggestion 2: It might make sense to support both R.id and R.string, similary to `assertDisplayed()`